.env.example

```
@@ -21,4 +21,13 @@ MIN_BOOKING_DAYS_AHEAD=0
 CHRISTMAS_END=2025-01-02
-CHRISTMAS_ACCESS_CODES=XMAS2024,VIP2024
\ No newline at end of file
+CHRISTMAS_ACCESS_CODES=XMAS2024,VIP2024
+
+# Email Configuration (SMTP)
+SMTP_HOST=hermes.utia.cas.cz
+SMTP_PORT=25
+SMTP_SECURE=false
+# SMTP_USER=    # Optional - leave empty for no auth
+# SMTP_PASS=    # Optional - leave empty for no auth
+EMAIL_FROM=noreply@chata.utia.cas.cz
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the SMTP_HOST key

(UnorderedKey)